runtime/src/lib.rs

```diff
@@ -440,6 +440,7 @@ impl pallet_encointer_ceremonies::Config for Runtime {
 impl pallet_encointer_communities::Config for Runtime {
 	type Event = Event;
 	type CommunityMaster = EnsureRoot<AccountId>;
+	type TrustableForNonDestructiveAction = EnsureRoot<AccountId>;
```

Comment:
This is meant to be 'EnsureSigned' AKA 'anyone who pays fees' because in the case of solochains (Gesell) we want to be permissive

Reply:
Please test in CI e2e that anyone can register a community

Reply:
oh ok, i wasn't sure about that and i wanted to be conservative with security :) does that generally mean that everything in `node/runtime` will only ever be run on testnets?

Reply:
Currently yes

Reply:
This is true only for the node though. The parachain is productive by default.